### PR TITLE
Added coverage and removed if statement in transformRegistry()

### PR DESCRIPTION
**Pull request by daveedgarcia**
Brought coverage up.

### CODE REVIEW

File: src/eureka-client.js [hunk 1]
```diff
@@ -204,18 +205,16 @@ export class Eureka {
     if (!registry) {
       throw new Error('Unable to transform empty registry');
     }
-    if (registry.length) {
```
Comment (jquatier) [inline src/eureka-client.js]: You know I added this for a reason, right?

Comment (daveedgarcia) [inline src/eureka-client.js]: Why is that? I have only seen registry as an object.

Comment (jquatier) [inline src/eureka-client.js]: Well, now that I've taken another look, it's silently broken anyway. When there's no instances `applications.application` is undefined, when theres 1 application, `applications.application` is an object, AND when there's more than 1 application, `applications.application` is an array. Can you change the `if` statement to `registry.applications.application` and then add another check for `registry.applications.application.length` and process a single application if that's undefined?

Comment (daveedgarcia) [inline src/eureka-client.js]: Sounds good. I'll make the updates later this evening.

Comment (jquatier) [inline src/eureka-client.js]: Sweet. I'll push a patch version to npm after that so we have a working module on npm again :P

Comment (daveedgarcia) [inline src/eureka-client.js]: Updated.

**Comment by jquatier** [comment]
Nice tests!

**Pull request merged**